README.md

```diff
@@ -212,6 +212,18 @@ At this stage your instance should be booting and become accessible shortly. Not
 Now go to your Obelisk project directory (`~/code/myapp`), and initialize a deployment config (`~/code/myapp-deploy`):
 Your project directory must be "thunkable", i.e. something on which `ob thunk pack` can be called. Usually it will be a git repository whose current revision has been pushed upstream.
 
+An example set of git commands to do this is as follows (Github):
+```bash
+git init
+git add .
+git commit -m "First Commit!"
+git remote add origin git@github.com:username/repo.git
+git push
```

Comment:
I had to run
```
git push --set-upstream origin master
```
I also had to create the repo on my github account (that may be obvious enough it doesn't need to be in the docs, but it could be worth an extra line)